ivadomed/utils.py

```diff
@@ -449,7 +449,7 @@ def segment_volume(folder_model, fname_image, gpu_number=0, options=None):
     # Load model training config
     context = imed_config_manager.ConfigurationManager(fname_model_metadata).get_config()
 
-    if any(pp in options for pp in ['thr', 'largest', ' fill_holes', 'remove_small']):
+    if options is not None and any(pp in options for pp in ['thr', 'largest', ' fill_holes', 'remove_small']):
```

Comment:
@andreanne-lemay : So.. `['thr', 'largest', ' fill_holes', 'remove_small']` are the param names from the SCT API and in the herebelow lines you are doing a matching between the ivadomed param names (ie in the config file) and these param names. Nothing crucial: but I would suggest to make this matching in SCT instead, so that these param names are consistent through `ivadomed`: what do you think? :-)

Reply:
The other user case of this function is when using `command=segment` but in that case we are using a config file so the param names are matching well (thanks to our new Config manager <3 ).

Reply:
If I understand well you are suggesting to rename 'thr' and 'largest'  to 'binarize_prediction' and 'keep_largest'  in SCT so all names are consistent (in SCT and ivadomed)? If so, I agree with this change. I opened an issue to fix this inconsistency in another PR: https://github.com/ivadomed/ivadomed/issues/506

Reply:
to be consistent with 'remove_small' maybe we should use the terminology 'keep_large' instead of 'keep_largest'?

also, i like 'thr' in that it is intuitive that a non-binary answer is expected (ie: the threshold), whereas when i see "binarize_prediction" I am tempted to assign a binary response (true/false). How about 'threshold_prediction'?

Reply:
Yes, I meant that it would be great if the names are consistent in SCT and ivadomed. So zuper for #506.
But.. it may not be always possible (for X reason, for instance with `thr`) and in that case I was suggesting to move at these lines do (ie match SCT_keyname and ivadomed_keyname) in the SCT function instead of in this ivadomed function (happy to re explain via phone :-) ).

Re. the specific case of `thr` vs `binarize_prediction` vs `threshold_prediction`: I am quite happy any ways: Just wanted to bring on the table the following info: this threshold in `ivadomed` currently refers to a threshold on the soft predictions. We are also developing (and soon into production?) a postprocessing which thresholds the soft predictions using the uncertainty map --> just to keep in mind when naming `thr` as we may have soon another threshold which uses the uncertainty map.